buildpipeline/DotNet-Trusted-Publish-Symbols.json

```diff
@@ -1,5 +1,29 @@
 {
   "build": [
+    {
+      "environment": {},
+      "enabled": true,
+      "continueOnError": true,
+      "alwaysRun": false,
+      "displayName": "Run AgentTools/Begin.ps1",
+      "timeoutInMinutes": 0,
+      "condition": "succeededOrFailed()",
```

Comment:
After reading https://docs.microsoft.com/en-us/vsts/build-release/concepts/process/conditions?view=vsts I suggest we use condition "always" for both begin and end scripts.